explainaboard/processors/processor.py

```diff
@@ -451,7 +456,9 @@ def get_overall_statistics(
         # declare customized features: _features will be updated
         custom_features: dict = metadata.get('custom_features', {})
         custom_analyses: list = metadata.get('custom_analyses', [])
-        metric_configs: dict = metadata.get('metric_configs', {})
+
+        metric_configs: dict = {"example": metadata.get('metric_configs', [])}
```

Comment:
Annotate full type information. Partial typing is not useful to detect errors.
```suggestion
        metric_configs: dict[str, list[MetricConfig]] = {"example": metadata.get('metric_configs', [])}
```